plugins/writeopia_persistence_room/src/commonMain/kotlin/io/writeopia/sdk/persistence/dao/room/RoomDocumentRepository.kt

```
@@ -44,7 +44,7 @@ class RoomDocumentRepository(
     ): List<Document> =
         emptyList()
 
-    override suspend fun deleteDocumentByFolder(folderId: String) {
+    override suspend fun deleteDocumentByFolder(folderId: String, workspaceId: String) {
     }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)